tor/core/inbox.py

```diff
@@ -76,7 +76,6 @@ def process_reply(reply, cfg):
     try:
         if any([regex.search(reply.body) for regex in MOD_SUPPORT_PHRASES]):
             process_mod_intervention(reply, cfg)
-            reply.mark_read()
             return
```

Comment:
Similar here, instead of `return`ing every time, it could be a lot cleaner using if-elif?

Reply:
Meh, this was just a code style thing to say "before anything else is processed, do any of these apply?" sort of block. It doesn't really matter to me all that much though. 👍